index.html

```diff
@@ -66,6 +64,188 @@ <h2>I’am a Software Developer</h2>
+              </p>
+
+              <ul>
+                <li>html</li>
+                <li>bootstrap</li>
+                <li>Ruby</li>
+              </ul>
+
+              <button class="button" type="button">See Project</button>
+            </div>
+
+            <div class="work-card">
+              <h2>Professional Art Printing Data</h2>
+              <p>
+                A daily selectionof privately personalized reads; no account or
+                sign-ups required.has been the industry's standard
+              </p>
+
+              <ul>
+                <li>html</li>
+                <li>bootstrap</li>
+                <li>Ruby</li>
+              </ul>
+
+              <button class="button" type="button">See Project</button>
+            </div>
+
+            <div class="work-card">
+              <h2>Professional Art Printing Data</h2>
+              <p>
+                A daily selectionof privately personalized reads; no account or
+                sign-ups required.has been the industry's standard
+              </p>
+
+              <ul>
+                <li>html</li>
+                <li>bootstrap</li>
+                <li>Ruby</li>
+              </ul>
+
+              <button class="button" type="button">See Project</button>
+            </div>
+
+            <div class="work-card">
+              <h2>Professional Art Printing Data</h2>
+              <p>
+                A daily selectionof privately personalized reads; no account or
+                sign-ups required.has been the industry's standard
+              </p>
+
+              <ul>
+                <li>html</li>
+                <li>bootstrap</li>
+                <li>Ruby</li>
+              </ul>
+
+              <button class="button" type="button">See Project</button>
+            </div>
+
+            <div class="work-card">
+              <h2>Professional Art Printing Data</h2>
+              <p>
+                A daily selectionof privately personalized reads; no account or
+                sign-ups required.has been the industry's standard
+              </p>
+              <ul>
+                <li>html</li>
+                <li>bootstrap</li>
+                <li>Ruby</li>
+              </ul>
+              <button class="button" type="button">See Project</button>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section class="about-me">
+        <div class="about-text">
+          <h2>About me</h2>
+          <p>
+            Hello I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don’t
+            hestiate to contact me.
+          </p>
+          <button class="about-button" type="button">Get my Resume</button>
+          .
+          <img class="side-image2" src="Icons/side_img2.png" alt="" />
```

Comment:
- [ ] Please The list of icons uses a `<ul>` tag to group them, and `<li>` per icon; each icon has a link (`<a>` tag)